### DISCUSSION TITLE
2024 Q3 OKR 손창엽

### DISCUSSION THREAD

**Discussion by soncy0311** [Pseudo-Lab/growth-collector#10] [우리들의 성장 목표!]
Objective: Persona AI 완성시키기

Key Result 1: sLLM 모델로 한국어 파인튜닝 진행해보기
Key Result 2: Persona를 적용한 LLM 서비스를 위한 논문 2개 읽고 리뷰
Key Result 3: sLLM 모델을 활용한 챗봇 개발해보기

**Comment by soncy0311** [comment]
Action Item
1. 페르소나를 검증할 수 있는 데이터셋 만들기
2. 페르소나 AI 관련 기업 3개, 그 기업의 인물 1명씩 조사

**Reply by yunhee1** [reply]
파이팅입니다-!

**Reply by seriousran** [reply]
이번 2주간은 꼭 액션을 취해봅시다!! :)

**Reply by leh60245** [reply]
화이팅입니다!!!!👍👍👍👍

**Reply by ZeVicTech** [reply]
같이 파이팅입니다!!

**Comment by soncy0311** [comment]
2024.10.23 (수) - 2024.10.27 (일)
이번주 Action Item
- 새로운 OKR 작성하기
    - 조건1: 장기적으로 연속성 있게 진행할 수 있는 것
    - 조건2: 성장에 도움이 될 수 있는 것
    - 조건3: 개인 브랜딩에도 도움이 될 수 있는 것
- 개인의 목표를 세우는데 도움이 될만한 컨텐츠 1개 읽고 공유하기

**Reply by yunhee1** [reply]
파이팅입니다 창엽님! ㅎㅎ :)

**Reply by SunandBean** [reply]
창엽님 파이팅입니다!!!

**Comment by soncy0311** [comment]
## 새로운 OKR
### Objective
Trend를 놓치지 않고 꾸준히 성장하는 개발자 되기

### Key Result
- KR1. 자동화 툴(Make or n8n)을 활용한 주식 종목 추천 봇 만들기
- KR2. 일주일에 1번 이상 경제 이슈 정리하기
- KR3. 사용해보지 않은 기술을 사용한 웹 뷰 서비스 개발 (위의 2개를 해낸다면 추가 도전,,,)

### Action Item 2024.11.11 - 2024.11.14
- 11월 1일 ~ 11월 10일까지의 경제 이슈 정리하기
- 최근 LLM에서 핫한 키워드 3개이상 정의하기

너무 늦어서 이번주 Action Item은 간단하게 작성했습니다....

**Reply by SunandBean** [reply]
창엽님 화이팅입니다!! 제일 큰 경제 이슈는 아무래도 트럼프 당선이 아닐까싶네요...ㅋㅋㅋ

**Reply by soncy0311** [reply]
저의 주제를 스포하시다니ㅎㅎㅎㅎ

**Comment by soncy0311** [comment]
### Action Item 2024.12.03 - 2024.12.12
- 경제 키워드 및 AI 키워드 3개씩 정리하기
- 아비투스 다 읽기
- 검색결과 정리 봇 만들기 (Google 위주)

**Reply by soncy0311** [reply]
이번 액션아이템은 2주치로 잡았습니다!
